Docs: metrics and alerts

jenmalloy opened the pull request:
### Type of change
- Documentation

### Description

Document metrics and alerts

### Checklist
- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [X] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

robshelly commented on documentation/modules/ref-rules-records.adoc:
```diff
@@ -0,0 +1,39 @@
+----
+up{job="address-space-controller"} or on(namespace) (1 - absent(up{job="address-space-controller"}))
+up{job="api-server"} or on(namespace) (1 - absent(up{job="api-server"}))
+----
```
The other examples and expressions in this module are formatted differently. Maybe they should be formatted ins this style too.

robshelly commented:
Since this issue was opened we have removed the API server so the ref-metrics-api-server.adoc can be removed.

jenmalloy commented:
@robshelly, thanks for your review! Addressed your comments (and also added this content to the existing monitoring assembly). Can you please review again? Thanks!

robshelly commented on documentation/assemblies/assembly-metrics-rules.adoc:
```diff
@@ -0,0 +1,15 @@
+
+include::../modules/ref-metrics-standard-controller-agent.adoc[leveloffset=+1]
+
+include::assemblies/assembly-rules.adoc[leveloffset=+1]
```
include::./assembly-rules.adoc[leveloffset=+1]

robshelly commented on documentation/assemblies/assembly-monitoring.adoc:
```diff
@@ -19,5 +19,7 @@ include::../modules/proc-deploy-monitoring-using-ansible.adoc[leveloffset=+1]
 
 include::../modules/proc-configure-alerts.adoc[leveloffset=+1]
 
+include::assemblies/assembly-metrics-rules.adoc[leveloffset=+1]
```
include::./assembly-metrics-rules.adoc[leveloffset=+1]

jenmalloy commented:
@robshelly, Sorry about that, I think I've got it fixed now: http://file.rdu.redhat.com/~jmalloy/openshift/#assembly-metrics-rules-messaging
